documentation/API_documentation/Carrier_Billing_Payment_API.md

```diff
@@ -0,0 +1,241 @@
+| purchaseCategoryCode | Allows aggregators/partners to specify the actual payee | Optionally valued by requester |
+| channel | Channel where the order occurred | Optionally valued by requester |
+| taxAmount | The tax amount charged by the merchant if the charge has tax already included (number). This also provides an indicator to the downstream billing system | Optionally valued by requester |
+| serviceId | Identifier of service related to the payment | Optionally valued by requester |
+| productId | Identifier of product related to the payment | Optionally valued by requester |
+
+**Payment State engine**
+
+Follow schema provides information about the payment state engine (state description & transition)
+
+<img src="./resources/Carrier_Billing_Payment_State_Engine.JPG" alt="PaymentState" title="Carrier Billing Payment API state engine">
+
+
+### 4.3 Endpoint Definitions
+
+Following table defines API endpoints of exposed REST based for Carrier Billing Payment API management operations.
+
+| **Endpoint** | **Operation** | **Description** |
+| -------- | --------- | ----------- |
+| POST<br>  \<base-url>/payment/v0/payments | **Create one step payment** | Create request for one step payment |
+| POST<br>  \<base-url>/payment/v0/payments/prepare | **Create two steps payment** | Create request for two steps payment |
+| POST<br>  \<base-url>/payment/v0/payments/confirm/{paymentId} | **Confirm a two steps payment** | Confirmation request for a prepared payment |
+| POST<br>  \<base-url>/payment/v0/payments/cancel/{paymentId} | **Cancel a two steps payment** | Cancellation request for a prepared payment |
+| GET<br> \<base-url>/payment/v0/payments | **Query for payment** | Querying for payment(s) with criteria |
+| GET<br> \<base-url>/payment/v0/payments/{paymentId} | **Retrieve payment** | Retrieve payment |
+<br>
+
+#### Carrier Billing Payment Resource Operations:
+<br>
+
+| **Create payment resource (one step)** |
+| -------------------------- |
+| **HTTP Request**<br> POST \<base-url>/payment/v0/payments<br>**Query Parameters**<br> No query parameters are defined.<br>**Path Parameters**<br> endUserId must be provided <br>**Request Body Parameters**<br> See above table for attribute definition.<br> Following attributes are mandatory in the request: endUserId; paymentAmount.chargingInformation.amount; paymentAmount.chargingInformation.currency ; paymentAmount.chargingInformation.description ; referenceCode.<br>Following attributes are only valued on server side and not present in POST request: transactionOperationStatus; resourceURL ; serverReferenceCode.
+
+ <br>**Response**<br> **201: payment created**<br>  Response body: A complete payment representation as described in above table is provided. <br> **400:** **Invalid input.**<br> **401:** **Un-authorized. <br> **403:** Forbidden.**<br> **409:** **Conflict.**<br> **500:** **Server Error.**<br> **503:** **Service temporarily unavailable.** |

```

Comment:
Think Ludovic "|" can be delete (formatting topic)

Reply:
Thanks @PedroDiez 
Done.